smspdudecoder/elements.py

```diff
@@ -127,7 +127,7 @@ def decode(cls, data: str) -> str:
         '0123456789'
         """
         data = swap_nibbles(data)
-        if data[-1:] == 'F':
+        if data[-1:] == 'F' or data[-1:] == 'f':
```

Comment:
Thank you for your contribution.

The same condition, in a slightly more canonical way:

```suggestion
        if data[-1:].lower() == 'f':
```

I can create a new release tomorrow.

By the way, in which use case did you encounter the lowercase "f"?

Reply:
Hahaha you're right, .lower() fits better.

I'm working on my own sms gateway with pyserial on a huawei e3372-325 modem. And im just getting pdu with f lowercase in the number section. 

Just like in here:
`07918497908906F0040b918431274322f400004270826170008037557d9ebe7edfdde935885e3ebf41ee7abb2caf83e0f2b7b8ae2e83e6e932480fa2bec561d07cfd76d3c36bfafb1e1ebb00`

Reply:
I actually have run into this a few times on my own and used this fix locally. I know its a old PR but is there any way we can still get this merged and released? I also do not mind helping out if you are looking for maintainers.

Reply:
> Hahaha you're right, .lower() fits better.
> 
> I'm working on my own sms gateway with pyserial on a huawei e3372-325 modem. And im just getting pdu with f lowercase in the number section.
> 
> Just like in here: `07918497908906F0040b918431274322f400004270826170008037557d9ebe7edfdde935885e3ebf41ee7abb2caf83e0f2b7b8ae2e83e6e932480fa2bec561d07cfd76d3c36bfafb1e1ebb00`

Very interesting, how many SMS are you able to send a day without getting blocked on your SMS gateway?

Reply:
I'm not sure. My gateway main purpose is receiving sms rather than sending. I'm not sure about limits on sending, but I think it is not really limited by the network operator. What are you planning to do?